### PR TITLE
fix: remove cli useless logs

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
CLI shows useless logs.

**Solution:**
Remove CLI useless logs

**Related Issue:**
#1015 
https://github.com/seal-io/walrus/issues/1405

### CODE REVIEW

File: staging/utils/runtimex/runtime.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/seal-io/walrus/utils/log"
 )
 
-func init() {
+func Init() {
```
Comment (gitlawr) [inline staging/utils/runtimex/runtime.go:11]: It is implicitly invoked before. Any impact of this change? Why do we need the change in runner.go?

Comment (aiwantaozi) [reply staging/utils/runtimex/runtime.go:11]: This implicitly invoked will always print the log inside the Set function while using log package, like below:
`maxprocs        maxprocs: Leaving GOMAXPROCS=%s: CPU quota undefined`

Comment (gitlawr) [reply staging/utils/runtimex/runtime.go:11]: Can we change Printf to Debugf?

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by alexcodelf — APPROVED** [review]
LGTM

**Pull request merged**